docs/source/advanced.rst

```diff
@@ -258,43 +258,187 @@ of :py:class:`scikeras.wrappers.BaseWrappers`. Currently, they are:
+        loss__from_logits=True,  # BinaryCrossentropy(from_logits=True) & CategoricalCrossentropy(from_logits=True)
+        loss__label_smoothing=0.1,  # passed to each sub-item, i.e. `loss={o: l(label_smoothing=0.1) for o, l in loss.items()}`
+        loss__out2__label_smoothing=0.5,  # overrides the above, results in CategoricalCrossentropy(label_smoothing=0.5)
+    )
+
+With this parameter routing in place, you can now use hyperparameter tuning on each loss' parameters.
 
-With this parameter rounting in place, you can now use hyperparameter tuning on each loss' ``label_smoothing`` parameter individually.
+Metrics
++++++++
 
-Example: callback with routed parameters
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Metrics have similar semantics to losses, but multiple metrics per output are supported.
+Here are several support use cases:
 
 .. code:: python
 
-    import tensorflow as tf
-    from sklearn.datasets import make_classification
-    from scikeras.wrappers import KerasClassifier
+    from tensorflow.keras.metrics import BinaryAccuracy, AUC
+
+    clf = KerasClassifier(
+        ...
+        metrics=BinaryAccuracy,
+        metrics__threashold=0.65,
+    )
+    # or to apply multiple metrics to all outputs
+    clf = KerasClassifier(
+        ...
+        metrics=[BinaryAccuracy, AUC],  # both applied to all outputs
+        metrics__0__threashold=0.65,
+    )
+    # or for different metrics to each output
+    clf = KerasClassifier(
+        ...
+        metrics={"out1": BinaryAccuracy, "out2": [BinaryAccuracy, AUC]},
+        metrics__out1__threashold=0.65,
+        metrics__out2__0__threashold=0.65,
+    )
+    # assuming you ordered your outputs like [out1, out2], this is equivalent to
+    clf = KerasClassifier(
+        ...
+        metrics=[BinaryAccuracy, [BinaryAccuracy, AUC]],
+        metrics__0__threashold=0.65,
+        metrics__1__0__threashold=0.65,
+    )
 
-    X, y = make_classification(1000, 20, n_informative=10, random_state=0)
 
-    def get_clf(meta) -> tf.keras.Model:
-        n_features_in_ = meta["n_features_in_"]
-        n_classes_ = meta["n_classes_"]
-        model = tf.keras.models.Sequential()
-        model.add(tf.keras.layers.InputLayer((n_features_in_,)))
-        model.add(tf.keras.layers.Dense(n_classes_, activation="softmax"))
-        return model
+See the `Keras Metrics docs`_ for more details on mapping metrics to outputs.
+
+Callbacks
++++++++++
+
+Just like metrics and losses, callbacks support several syntaxes to compile them depending on your needs:
+
+.. code:: python
+
+    clf = KerasClassifier(
+        ...
+        callbacks=[tf.keras.callbacks.BaseLogger, tf.keras.callbacks.EarlyStopping]
+        callbacks__1__monitor="loss",
+    )
+    # or
+    clf = KerasClassifier(
+        ...
+        callbacks={"bl": tf.keras.callbacks.BaseLogger, "es": tf.keras.callbacks.EarlyStopping}
+        callbacks__es__monitor="loss",
+    )
+    # or for a single callback
```

Comment:
This example should be moved up. It's probably the most common use case. I might specify the list/dict examples in a separate code block.

Reply:
Donzo, thanks for the suggestion